xml/System.Windows.Forms/BindingContext.xml

```diff
@@ -200,7 +200,11 @@ this->BindingContext->get_Item(dataSet1->Customers)->Position = 1;
         <param name="dataSource">The <see cref="T:System.Object" /> associated with the <see cref="T:System.Windows.Forms.BindingManagerBase" />.</param>
         <param name="listManager">The <see cref="T:System.Windows.Forms.BindingManagerBase" /> to add.</param>
         <summary>Adds the <see cref="T:System.Windows.Forms.BindingManagerBase" /> associated with a specific data source to the collection.</summary>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+This method is obsolete and unused.
```

Comment:
These APIs are obsolete but they don't have an obsolete attribute?

Also, obsolete only in .NET Core? We'd need to tell since when because the docs are for all versions.